common/src/main/java/com/wynntils/gui/render/Texture.java

```diff
@@ -50,10 +50,11 @@ public enum Texture {
     SETTINGS_ICON("settings_icon.png", 17, 34),
 
     // Map related
-    GILDED_MAP_TEXTURES("map/gilded_map_textures.png", 262, 524),
-    PAPER_MAP_TEXTURES("map/paper_map_textures.png", 256, 512),
-    WYNN_MAP_TEXTURES("map/wynn_map_textures.png", 256, 256),
-    FULLSCREEN_MAP_BORDER("map/full_map_border.png", 510, 254),
+    GILDED_MAP_TEXTURES("map/map_border/gilded_map_textures.png", 262, 524),
+    PAPER_MAP_TEXTURES("map/map_border/paper_map_textures.png", 256, 512),
+    WYNN_MAP_TEXTURES("map/map_border/wynn_map_textures.png", 126, 256),
+    CIRCLE("map/map_border/circle.png", 256, 256),
```

Comment:
This CIRCLE is only used as a mask, right? I think the name (both of the enum and the file) should convey that.